perun-core/src/main/java/cz/metacentrum/perun/core/api/ResourcesManager.java

```diff
@@ -65,6 +65,21 @@
 	 */
 	Resource createResource(PerunSession perunSession, Resource resource, Vo vo, Facility facility) throws InternalErrorException, PrivilegeException, VoNotExistsException, FacilityNotExistsException, ResourceExistsException;
 
+	/**
+	 * Copy "template" settings(attributes,services,tags (if exists),...) from user's another existing resource and create new resource with this template.
```

Comment:
List of copied setting is very important. User need to know what settings will be copied and what is needed to be set manually. Please be more specific about it.